cfgov/jinja2/v1/_includes/organisms/email-popup.html

```diff
@@ -37,11 +37,11 @@
                 </div>
                 <p>
                   <a href="/owning-a-home/privacy-act-statement/" target="_blank">
-                    <span class="jump-link_text">View Privacy Act statement</span>
+                    <span class="a-link_text">View Privacy Act statement</span>
```

Comment:
Removed the span in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/f13808069fbcf7ded59ba886d5170effbef81280
Also, added a code comment of where this is used.